ui/admin/app/styles/app.scss

```diff
@@ -452,3 +454,103 @@
     }
   }
 }
+
+// secret editor
+.json-secret-editor {
+  .CodeMirror {
+    height: 210px;
+  }
+
+  .CodeMirror-gutters {
+    background: #31332b;
```

Comment:
Do we need these hard coded colors? Does our current color scheme not work?

Reply:
This color was added based off the Monokai color scheme. It was needed primarily for creating the editor skeleton. I was going to ask the team and @joshuaogle if I should created a color variable for this which I am happy to do. I was not exactly sure where best to declare that though.